src/sentry/receivers/sentry_apps.py

```diff
@@ -31,14 +32,37 @@ def issue_saved(sender, instance, created, **kwargs):
     )
 
 
+@issue_assigned.connect(weak=False)
+def issue_assigned(project, group, user, **kwargs):
```

Comment:
Will re-using this name effect the signal object at all?

Reply:
Heh, I don't know. Didn't even think about that, tbh. I don't know if I can Python well enough to know. I'm going to change one of them to be safe.

Reply:
wait @markstory did you mean reusing the signal in other places or having the method be the same as the signal? I feel like it makes more sense to change the method name than the signal name if thats the case

Reply:
I meant that the signal and function have the same name, and python modules are basically dictionaries at the end of the day, so one will overwrite the other.

Reply:
cool. So I'm in favor of renaming the function to something like `send_issue_assigned_webhook` and keeping the signal as `issue_assigned` so that its clear what the function is doing (and also saves you from having to rename the import)

Reply:
Works for me. 👍